src/home.html

```diff
@@ -328,8 +328,8 @@ <h1 class="slider_content-title">
                 <!-- project -->
                 <div class="project row space-top">
                     <div class="col c-6 m-12 s-12">
-                        <div class="project-img" data-bg-image="./assets/img/background.jpg">
-                            <img src="" alt="">
+                        <div class="project-img" data-bg-image="">
```

Comment:
@vihan85 
Are you use "data-bg-image"? 
If you don't use it, you can delete it, please
Check all sites as the same issue.